packages/api/src/external/fhir-converter/converter.ts

```diff
@@ -27,7 +27,7 @@ export enum FHIRConverterCDATemplate {
 
 export type ContentMimeType = Pick<Document["content"], "mimeType">;
 
-export function isConvertible(mimeType?: string): boolean {
+export function isConvertible(mimeType?: string | null): boolean {
```

Comment:
Can we use `undefined`? We only use `null` as a type for HTTP and DB interfaces.